pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+requires = ["poetry-core>=1.1.0b3"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "trino-python-client"
+version = "0.314.0"
```

Comment:
We need some changes in versioning guides in README.md.

Reply:
@hovaesco could you provide more context about what is required. I merely lifted this from [here](https://github.com/trinodb/trino-python-client/blob/master/trino/__init__.py#L22) which doesn't seem to be referenced elsewhere.

Reply:
Please take a look [here](https://github.com/trinodb/trino-python-client/blob/master/README.md?plain=1#L486)